simple/testdata/make-lencap.go

```diff
@@ -0,0 +1,17 @@
+package pkg
+
+func fn() {
+	const c = 0
+	var x, y int
+	type s []int
+	_ = make([]int, 1)
+	_ = make([]int, c)       // constant of 0 maybe due to debugging, math or platform-specific code
```

Comment:
Don't use `[]int` to test correct handling of constants, as it will never complain about slices, anyway.